LICENSE

```diff
@@ -25,3 +25,13 @@ INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
 CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 POSSIBILITY OF SUCH DAMAGE.
+
+--------------------------------------------------------------------------------
+
+GlobalPlatform API license
+
+Since OP-TEE is a TEE implementing GlobalPlatform APIs, users of OP-TEE should
+get the license for the GlobalPlatform TEE API either by joining GlobalPlatform
+as a member or, they can get it for free by going to and download the
```

Comment:
From the [GP website](https://www.globalplatform.org/specificationform.asp) (I bold an interesting sentence):
> Upon completion of the contact information and indication of your acceptance of the License Agreement, all interested parties are welcome to download the GlobalPlatform Specifications. **Please note that this license covers both evaluation and product development**. As a result, you do not need to complete a new License Agreement if you decide to go on to develop and sell a product using these specifications. All rights granted under the License Agreement are provided at no charge.

These "contact information and "indication of your acceptance" are requested when one download any of the GPD TEE specifications.